test/src/wd-manager/testWebdriver.js

```diff
@@ -1,5 +1,6 @@
 const assert = require('assert');
 const mockery = require('mockery');
+const {instances} = require('../../../lib/runner/webdriver-server.js');
```

Comment:
can we use `WDServer.instances` instead of importing it separately?

Reply:
thanks for catching this, removed as wasn't using it